adminer/drivers/elastic.inc.php

```diff
@@ -292,15 +301,22 @@ function tables_list() {
 
 	function table_status($name = "", $fast = false) {
 		global $connection;
+<<<<<<< HEAD
 		$search = $connection->query("_search?search_type=count", array(
+=======
+
+		$search = $connection->query("_search", array(
+			"size" => 0,
+>>>>>>> elastic
```

Comment:
You've committed a merge conflict.

Reply:
Sorry! I think I messed .travis.yml also, so I reverted it, but adding two commands on script section to check also the compiled file. Because on first of my commits, the code passed on linter, but compiler could not generate a good compiled file.